### ISSUE TITLE
get_account_history limit vs. actual number of ops received

### ISSUE THREAD

**Issue by crokkon** [holgern/beem#267]
Sorry, I haven't found the time to address this myself, leaving it here in case you or sb else has capacity and interest to get this compatible with Blurt:

`Account.get_account_history(start, limit=1000)` returns 1001 ops on Hive/Steem, but 1000 on Blurt. The assumption of getting`limit + 1` ops is currently hard coded into `Account.history()` / `Account.history_reverse()`, e.g.
https://github.com/holgern/beem/blob/2c91a7e6d476c0ace7835344ee09dc0280284b30/beem/account.py#L2275
https://github.com/holgern/beem/blob/2c91a7e6d476c0ace7835344ee09dc0280284b30/beem/account.py#L2279
As a result, the account history ops returned on Blurt are missing every 1001st op.

My first though was setting the start op for the next iteration based on the actual number of ops received instead of `_limit + 1`, but that doesn't go well with Blurt where the account history is truncated and may return empty lists for early op ranges.

**Comment by crokkon** [comment]
```python
from beem import Blurt
from beem.account import Account
import sys

blurt = Blurt('https://rpc.blurt.world')
account = Account("eastmael", blockchain_instance=blurt)
idx = 0
for op in account.history():
    sys.stdout.write("%s\r" % (idx))
    if idx != op['index']:
        print("Expected op %d, got %d" % (idx, op['index']))
        idx = op['index']
    idx+= 1
```
```
Expected op 0, got 24259
Expected op 25025, got 25026
Expected op 26026, got 26027
Expected op 27027, got 27028
Expected op 28028, got 28029
Expected op 29029, got 29030
Expected op 30030, got 30031
Expected op 31031, got 31032
Expected op 32032, got 32033
Expected op 33033, got 33034
Expected op 34034, got 34035
Expected op 35035, got 35036
Expected op 36036, got 36037
Expected op 37037, got 37038
Expected op 38038, got 38039
...
```

**Comment by crokkon** [comment]
fixed with a905e3f - thank you!
